DBugVision/.idea/DBugVision2017.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This, and all the other files under `DBugVision/.idea/` should not be pushed into git. Remove them from here, and update the `.gitignore` file.